skabase/SKATestDevice/SKATestDevice/SKATestDevice.py

```diff
@@ -179,6 +213,78 @@ def RunGroupCommand(self, argin):
         return argout
         # PROTECTED REGION END #    //  SKATestDevice.RunGroupCommand
 
+    def devLogMsg(self, devlogmsg, devloglevel):
+
+        if self._element_logging_level >= 1 and devloglevel == 1:
+            self.fatal_stream("%s", devlogmsg)
```

Comment:
Think this could be simplified to `self.fatal_stream(devlogmsg)`.

Reply:
Corrected.